packages/core/src/status-adornment/StatusAdornment.css

```diff
@@ -2,8 +2,8 @@
   align-self: center;
   display: inline-block;
   fill: var(--saltStatusAdornment-color, var(--statusAdornment-color));
-  height: var(--salt-icon-size-status-adornment);
-  min-height: var(--salt-icon-size-status-adornment);
+  height: var(--salt-size-adornment);
+  min-height: var(--salt-size-adornment);
```

Comment:
is size-adornment used for anything else other than icon - If not then why then name change ?

Reply:
i think designs have been updating names on their end and code hasnt kept up so this PR is to ensure they both align